### PR TITLE
Update windows.md

### PR DESCRIPTION
**Pull request by boofinka**
Added infobox to address NXLog multiline parsing.

### CODE REVIEW

File: _source/logzio_collections/_log-sources/windows.md [hunk 1]
```diff
@@ -145,6 +145,9 @@ LogFile %ROOT%\\data\\nxlog.log
 </Extension>
 ```
 
+  See the [NXLog Documentation](https://nxlog.co/documentation/nxlog-user-guide/parsing-multiline.html#parsing-multiline) for more information on configuring multiline parsing og NXLog logs.
```
Comment (boofinka) [inline _source/logzio_collections/_log-sources/windows.md]:
```suggestion
See the
[NXLog Documentation](https://nxlog.co/documentation/nxlog-user-guide/parsing-multiline.html#parsing-multiline)
for more information on configuring multiline parsing of NXLog logs.
```

Comment (boofinka) [inline _source/logzio_collections/_log-sources/windows.md]:
```suggestion
See the
[NXLog Documentation](https://nxlog.co/documentation/nxlog-user-guide/parsing-multiline.html#parsing-multiline)
for more information on configuring multiline parsing of NXLog logs.
```

File: _source/logzio_collections/_log-sources/windows.md [hunk 2]
```diff
@@ -145,6 +145,9 @@ LogFile %ROOT%\\data\\nxlog.log
...
+  {:.info-box.read}
```
Comment (boofinka) [inline _source/logzio_collections/_log-sources/windows.md]:
```suggestion
{:.info-box.read}
```

File: _source/logzio_collections/_log-sources/windows.md [hunk 3]
```diff
@@ -145,6 +145,11 @@ LogFile %ROOT%\\data\\nxlog.log
 </Extension>
 ```
 
+See the
+[NXLog Documentation](https://nxlog.co/documentation/nxlog-user-guide/parsing-multiline.html#parsing-multiline)
+for more information on configuring multiline parsing of NXLog logs.
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/windows.md]: We're parsing Windows logs here, right? If you don't specifically mean logs generated by NXLog, I'd suggest revising to something like this:

```suggestion
for more information on parsing multiline logs.
```

Comment (boofinka) [reply _source/logzio_collections/_log-sources/windows.md]: I don't have any objections.

**Pull request merged**